src/main/java/com/checkmarx/flow/config/FlowProperties.java

```diff
@@ -311,7 +311,9 @@ public void setThresholds(Map<FindingSeverity, Integer> thresholds) {
         private Integer port = 25;
         private String username;
         private String password;
-        private boolean enabled = false;
+        private List<String> cc;
+        private boolean notification = false;
```

Comment:
isNotificationEnabled or notificationEnabled would be a better name.

Reply:
Renamed to notificationEnabled.